bundle/manifests/nic-configuration-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,373 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "configuration.net.nvidia.com/v1alpha1",
+          "kind": "NicConfigurationTemplate",
+          "metadata": {
+            "name": "connectx6-config",
+            "namespace": "nic-configuration-operator"
+          },
+          "spec": {
+            "nicSelector": {
+              "nicType": "101b",
+              "pciAddresses": [
+                "0000:03:00.0",
+                "“0000:04:00.0”"
+              ],
+              "serialNumbers": [
+                "MT2116X09299"
+              ]
+            },
+            "nodeSelector": {
+              "feature.node.kubernetes.io/pci-15b3.present": "true"
+            },
+            "resetToDefault": false,
+            "template": {
+              "gpuDirectOptimized": {
+                "enabled": true,
+                "env": "Baremetal"
+              },
+              "linkType": "Ethernet",
+              "numVfs": 2,
+              "pciPerformanceOptimized": {
+                "enabled": true,
+                "maxAccOutRead": 44,
+                "maxReadRequest": 4096
+              },
+              "roceOptimized": {
+                "enabled": true,
+                "qos": {
+                  "pfc": "0,0,0,1,0,0,0,0",
+                  "trust": "dscp"
+                }
+              }
+            }
+          }
+        },
+        {
+          "apiVersion": "configuration.net.nvidia.com/v1alpha1",
+          "kind": "NicDevice",
+          "metadata": {
+            "name": "co-node-25-101b-mt2232t13210",
+            "namespace": "nic-configuration-operator"
+          },
+          "spec": {
+            "configuration": {
+              "template": {
+                "linkType": "Ethernet",
+                "numVfs": 8,
+                "pciPerformanceOptimized": {
+                  "enabled": true
+                }
+              }
+            }
+          },
+          "status": {
+            "conditions": [
+              {
+                "reason": "UpdateSuccessful",
+                "status": "False",
+                "type": "ConfigUpdateInProgress"
+              }
+            ],
+            "firmwareVersion": "20.42.1000",
+            "node": "co-node-25",
+            "partNumber": "mcx632312a-hdat",
+            "ports": [
+              {
+                "networkInterface": "enp4s0f0np0",
+                "pci": "0000:04:00.0",
+                "rdmaInterface": "mlx5_0"
+              },
+              {
+                "networkInterface": "enp4s0f1np1",
+                "pci": "0000:04:00.1",
+                "rdmaInterface": "mlx5_1"
+              }
+            ],
+            "psid": "mt_0000000225",
+            "serialNumber": "mt2232t13210",
+            "type": "101b"
+          }
+        }
+      ]
+    capabilities: Basic Install
```

Comment:
Missing annotations:
`description`
all the `features.operators.openshift.io`
`operatorframework.io/suggested-namespace`
`provider`
`repository`
`support`
See:
https://github.com/redhat-openshift-ecosystem/certified-operators/blob/main/operators/nvidia-network-operator/24.10.0/manifests/nvidia-network-operator.clusterserviceversion.yaml#L91-L109